npm_and_yarn/lib/dependabot/npm_and_yarn/file_updater/npm_lockfile_updater.rb

```diff
@@ -256,6 +236,41 @@ def flattenend_manifest_dependencies
             end
         end
 
+        # Runs `npm install` with `--package-lock-only` flag to update the
+        # lockfiile.
+        #
+        # Other npm flags:
+        # - `--force` ignores checks for platform (os, cpu) and engines
+        # - `--dry-run=false` the updater sets a global .npmrc with dry-run:
+        #   true to work around an issue in npm 6, we don't want that here
```

Comment:
I'd treat `dry-run: true` as a single word and not break across lines in the middle of that to reduce risk of misreading.

Reply:
You mean quoting it, right?

Reply:
I was thinking:
```
        # - `--dry-run=false` the updater sets a global .npmrc with
        #    dry-run: true to work around an issue in npm 6, we don't want that here
```

But quoting it is probably even better
```
        #    `dry-run: true` to work around an issue in npm 6, we don't want that here
```

Regardless of quoting or not, I still wouldn't break across lines... too easy for the eye to miss the quote if partially split across lines.

Reply:
Aaah I see, fixed!

Reply:
# - [ ] ~~````

- - [ ] # > ````~~~~@![Screenshot_20241216-143552.jpg](https://github.com/user-attachments/assets/25d71c81-4361-49c9-9057-062fdb88ff58)

 ~~